modules/otel-config-multicluster.adoc

```diff
@@ -106,21 +106,26 @@ spec:
     exporters:
       otlphttp:
         endpoint: https://observability-cluster.com:443 # <1>
-        insecure: false
-        compression: on
         tls:
-          cert_file: "/path/to/server-cert.pem"
-          key_file: "/path/to/server-key.pem"
-          client_ca_file: "/path/to/client-ca.pem"
-
+          insecure: false
+          cert_file: /certs/server.crt
+          key_file: /certs/server.key
+          ca_file: /certs/ca.crt
     service:
       pipelines:
         traces:
           receivers: [jaeger, opencensus, otlp, zipkin]
           processors: [memory_limiter, k8sattributes, resourcedetection, batch]
           exporters: [otlp]
+  volumes:
+    - name: otel-certs
+      secret: 
+        name: otel-certs
+  volumeMounts:
+    - name: otel-certs
+      mountPath: /certs
 ----
-<1> The Collector exporter is configured to export OTLP HTTP and points to the OpenTelemetry Collector from the central cluster.
+<1> The Collector exporter is configured to export OTLP HTTP and points to the OpenTelemetry Collector from the central cluster. Note you need to provide the certificate as a secret and mount it in your OpenTelemetry Collector instance. You can create the certificate by hand or using the {cert-manager-operator}.
```

Comment:
@iblancasa, what kind of certificate is this about?

Reply:
This needs:
- A TLS cert (`cert_file`)
- A TLS key (`key_file`)
- A CA cert (`ca_file`)

Reply:
Is it possible that these certs will be already created by some users for other purposes?
Or do these certs have to be created for every installed OTEL instance?

Reply:
They could have create them before.

Reply:
@iblancasa, WDYT if I add this to the Prerequisites and if I delete the second sentence on line 128?
 
```
* You have the following certificates mounted as secrets in your {OTEL} instance:
+
[TIP]
====
You can create these certificates manually or by using the {cert-manager-operator}.
====
+
** TLS certificate, for example as a `cert_file`.
** TLS key, for example as a `key_file`.
** CA certificate, for example as a `ca_file`.
```

Reply:
LGTM